### PR TITLE
[CBRD-23842] Fix the bug when excute the trigger statement in CDC

### PR DESCRIPTION
**Pull request by joohok**
http://jira.cubrid.org/browse/CBRD-23842

Purpose

There is the bug when creating a trigger that has no target class. 
For example, `create trigger ... execute print` statement has no target class, but existing code try to access the target class.
So, the logic for checking if the target class exists is added

### CODE REVIEW

**Review by eido5 — APPROVED** [review]

File: src/parser/parse_tree.h [hunk 1]
```diff
@@ -3870,8 +3870,7 @@ enum cdc_ddl_type
   CDC_ALTER,
   CDC_DROP,
   CDC_RENAME,
-  CDC_TRUNCATE,
-  CDC_TRUNCATE_CASCADE
```
Comment (wrlawodms) [inline src/parser/parse_tree.h]: Why is this removed?

Comment (joohok) [reply src/parser/parse_tree.h]: It is unrelated change. 
I removed it because `TRUNCATE_CASCADE` is obsolete.
I will revert it, because it will be treated In other issue that add some more ddl types.

**Review by wrlawodms — APPROVED** [review]

**Review by hornetmj — APPROVED** [review]
